source/dubregistry/internal/utils.d

```diff
@@ -13,3 +23,56 @@ string black(string url)
 @safe {
 	return black(URL(url)).toString();
 }
+
+static immutable string logoOutputFolder = "uploads/logos";
+
+static immutable string[] logoFormats = [".png"];
+
+/// 
+/// Params:
+///   file = the file to convert
+///   name = the logo name to put in uploads/logos
+///   deleteExisting = if false, throw an exception if the files already exist
+///   deleteFinish = if true, delete the input file after at least one successful conversion
```

Comment:
Yeah, unfortnately it wasn't possible so far to persuade Walter into fixing the semantics for this. What this will do is to create one paragraph per line instead of a single one.